ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1068,3 +1068,33 @@ def test_tensorflow_zero_fraction(
         fn_tree="math.zero_fraction",
         value=x[0],
     )
+
+
+# truediv
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.math.truediv"
+    ),
+)
+def test_tensorflow_truediv(
+    dtype_and_x, as_variable, num_positional_args, native_array, fw
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
```

Comment:
The tests are failing due to `fw` keyword argument, `fw` was removed from the testing function and it's outdated now, could you remove that one? otherwise lgtm, thanks!